tidy3d/components/grid/mesher.py

```diff
@@ -254,23 +320,34 @@ def structure_steps(
             Wavelength to use for the step size and for dispersive media epsilon.
         min_steps_per_wvl : float
             Minimum requested steps per wavelength.
+        global_min_dl: pd.NonNegativeFloat
+            Lower bound of grid size.
+        axis : Axis
+            Axis index along which to operate.
         """
         min_steps = []
         for structure in structures:
-            n, k = structure.medium.eps_complex_to_nk(structure.medium.eps_model(C_0 / wavelength))
-            index = max(abs(n), abs(k))
-            min_steps.append(wavelength / index / min_steps_per_wvl)
+            min_step_i = global_min_dl
```

Comment:
remove this line and then use `global_min_dl` in 336 and 338?

Reply:
Removed `min_step_i` and append to `min_steps` directly.